neqo-http3-server/src/main.rs

```diff
@@ -335,6 +337,7 @@ fn init_poll(
     Ok((poll, sockets, servers))
 }
 
+#[allow(clippy::cognitive_complexity)]
```

Comment:
Please factor the code rather than suppress warnings.  The loop on curr_active seems to be easily extracted.